react/new/src/components/MainNavigation/index.tsx

```diff
@@ -1,9 +1,15 @@
 import { Menu, Group, Center, Burger, Container } from '@mantine/core'
 import { useDisclosure } from '@mantine/hooks'
-import { IconChevronDown } from '@tabler/icons-react'
+import {
+  IconChevronDown,
+  IconChevronRight,
+  IconSettings,
+} from '@tabler/icons-react'
 import { Link } from 'react-router-dom'
+import { useState } from 'react'
 import classes from './main-navigation.module.scss'
 import routes from '../../routes'
+import { ThemeControls } from '../ThemeControls'
```

Comment:
Is there a way to do absolute file paths from project root instead of relative? Also just trying to learn the language.